.github/workflows/build.yaml

```diff
@@ -171,3 +171,46 @@ jobs:
 
       - name: flake8
         run: python3 -m flake8
+
+  build_plenum_release:
+    name: Indy Plenum Release
+    needs: [indy_plenum, indy_plenum_tests, lint]
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/${{ needs.workflow-setup.outputs.GITHUB_REPOSITORY_NAME }}/plenum-build
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Install os dependencies
+        run: ./build-scripts/ubuntu-1604/install-os-dependencies.sh
+
+      - name: Install os tools
+        run: ./build-scripts/ubuntu-1604/install-os-tools.sh
+
+      - name: Build Indy Plenum deployment package
+        run: ./build-scripts/ubuntu-1604/build-indy-plenum.sh /__w/indy-plenum/indy-plenum 1.14.0 /tmp
+
+      - name: Build 3rd party deployment packages
+        run: ./build-scripts/ubuntu-1604/build-3rd-parties.sh
+
+  publish_plenum_rc:
+    name: Publish release candidate
+    runs-on: ubuntu-20.04
+    needs: ['build_plenum_release']
+    if: github.event_name == 'push' && (github.repository == 'ianco/indy-plenum' && github.ref == 'refs/heads/release*')
+    steps:
+      - name: pub
+        run: |
+          echo "publish rc"
+  publish_plenum_release:
+    name: Publish release
+    runs-on: ubuntu-20.04
+    needs: ['build_plenum_release']
+    if: github.event_name == 'push' && (github.repository == 'ianco/indy-plenum' && github.ref == 'refs/heads/master')
```

Comment:
Please remove the reference to your fork; `ianco/indy-plenum`.

Reply:
Done